webui/react/src/hooks/useMetricNames.ts

```diff
@@ -24,7 +24,7 @@ const useMetricNames = (
 
   useEffect(() => {
     if (actualExpIds.length === 0) {
-      setMetrics(NotLoaded);
+      setMetrics(Loaded([]));
```

Comment:
`NotLoaded` caused unwanted loading. `Loaded([])` is appropriate here since data validation is done (Loaded) and no data (`[]` empty array)